CMakeLists.txt

```diff
@@ -303,6 +303,7 @@ include_directories( ${elxINCLUDE_DIRECTORIES} )
 # to the ITK the BEFORE keyword is not needed anymore.
 if( ELASTIX_USE_OPENCL )
   include_directories( BEFORE ${elxCommon_OpenCL_INCLUDE_DIRECTORIES} )
+  list( APPEND elxINCLUDE_DIRECTORIES ${elxCommon_OpenCL_INCLUDE_DIRECTORIES} )
```

Comment:
It looks reasonable, but can you please tell us which problem is solved by adding `elxCommon_OpenCL_INCLUDE_DIRECTORIES` to `elxINCLUDE_DIRECTORIES`? Did you get any compilation error before?

Reply:
Yes this resolves missing include errors for the associated headers. It is required for [ITKElastix](https://github.com/InsightSoftwareConsortium/ITKElastix) or any other project that build against elastix with `ELASTIX_USE_OPENCL` enabled.